### DISCUSSION TITLE
RotateLabel Props angle

### DISCUSSION THREAD

**Discussion by aabidDevit** [Abhinandan-Kushwaha/react-native-gifted-charts#920] [Q&A]
Barchart -> I want the x-axis labels to be rotated at 90 degrees. Is there any way to set a degree?

![Screenshot_1733744964](https://github.com/user-attachments/assets/3df78120-45bb-4499-90d8-43aa07b07ad6)

**Comment by Abhinandan-Kushwaha** [comment]
yes @aabidDevit you can use custom X-axis labels using the `labelComponent` property instead of the `label` property in the data array.

Since `labelComponent` is your own custom component, you can rotate it using your own css styles.
